src/main/java/org/gridsuite/network/map/dto/utils/ElementUtils.java

```diff
@@ -86,9 +86,19 @@ public static Optional<ActivePowerControlInfos> toActivePowerControl(Identifiabl
                         .droop(activePowerControl.getDroop()).build());
     }
 
-    public static String toBranchStatus(Branch<?> branch) {
-        var branchStatus = branch.getExtension(OperatingStatus.class);
-        return branchStatus == null ? null : branchStatus.getStatus().name();
+    public static String toOperatingStatus(Identifiable<?> identifiable) {
```

Comment:
pass `Extendable` instead of `Identifiable` then 
```
if(identifiable instanceof Branch<?> || ThreeWindingsTransformer || HvdcLine) {
  var operatingStatus = extandable.getExtension(OperatingStatus.class);
  return operatingStatus == null ? null : operatingStatus.getStatus().name();
}
return null;
```